global.json

```diff
@@ -2,7 +2,7 @@
   "tools": {
     "dotnet": "5.0.100-preview.5.20251.2",
     "runtimes": {
-      "dotnet/x64": [
+      "dotnet": [
```

Comment:
Is this really the right syntax? I'm assuming arcade will detect the configuration/arch and download the appropriate runtime?